### PR TITLE
add image

### PR DESCRIPTION
**Pull request by Yexiansen66**

### CODE REVIEW

File: docs/TEAM.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
 | |[Aravind Sivasailam](https://github.com/aravindsivasailam) | I'm a product engineer living in US with 2 years of experience in software testing and project management. New to software development and wish to improve in HTML, CSS and JS. |
 | |[Edwin Mancipe](https://github.com/efmg0325)|I am from Colombia. I have a bit of knowledge about HTML, CSS, Javascript, Python and SQL. I love to listeninng to Heavy Metal music. I like second life web browser games.|
 | |[Edori Atiri](https://github.com/EdoriAtiri) | From Nigeria. Began to learn Web Development this year. I hope to learn and contribute while I'm here.|
-
+| <img src="https://avatars2.githubusercontent.com/u/58055517?s=400&u=7fc1b9106782f52a5b563a7ae4ca277ea00cd9c3&v=4" width="225px" height="100px"> | [Fusen Ye](https://github.com/Yexiansen66) | I'm a beginner from China. I know HTML,CSS and JavaScript. |
```
Comment (ni3-kdm) [inline docs/TEAM.md:37]: @Yexiansen66 Can you add your image in the `docs\images\team` directory and use the same here.

Comment (Yexiansen66) [reply docs/TEAM.md]: I changed.But I have a new error: Docker build failed with exit code 1.What should I do?

Comment (Yexiansen66) [reply docs/TEAM.md]: I changed.But I have a new error: Docker build failed with exit code 1.What should I do?

Comment (ni3-kdm) [reply docs/TEAM.md]: We haven't use docker yet. May be you can ping me on discord to check this issue.

**Comment by ni3-kdm** [comment]
@Yexiansen66 

You can just add `docs/TEAM.md` and `docs/images/team/fusen.jpg` in this PR. Other files are not needed. Also, Avoid making changes in the existing working code.

**Pull request closed without merging**